### PR TITLE
Add 'config' option for runner

### PR DESCRIPTION
**Pull request by jakubzapletal**

### CODE REVIEW

**Comment by jakubzapletal** [comment]
Hi @richardmiller , how does it look like with this PR? Anything wrong? (and related other 2 PRs)

**Comment by richardmiller** [comment]
Hi @jakubzapletal, sorry for the delay. Taking a proper look now.

File: src/PhpSpecExtension.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function load(ContainerBuilder $container, array $config)
         $loader->load('services.xml');
 
         $container->setParameter('phpspec_extension.path', $config['path']);
+        $container->setParameter('phpspec_extension.config', $config['config']);
```
Comment (richardmiller) [inline src/PhpSpecExtension.php]: I'm getting a warning about this index being missing as the default is null and I'm not setting it in the config file. Can it default to the usual location?

Comment (jakubzapletal) [inline src/PhpSpecExtension.php]: Sure

**Comment by jakubzapletal** [comment]
Hi @richardmiller , is it ok now?

**Comment by richardmiller** [comment]
Yes, great thank you

**Comment by jakubzapletal** [comment]
Cool, thanks

**Pull request merged**